Lizarov, Anna.Assignment#3.Rmd

```diff
@@ -0,0 +1,216 @@
+
+```
+
+```{r}
+#Standardizing values
+K3 <- scale(K3)
+```
+
+# 2 Clusters
+```{r}
+fit <- kmeans(K3, 2)
+fit$cluster
+(K4 <- data.frame(K3, fit$cluster))
+
+```
+
+```{r}
+names(K4) <- c("1", "2", "3", "4", "5", "Cluster")
+```
+
+   #Visualizing the 2 clusters. 
+```{r}
+K5 <- K4 %>% gather(Week,Scores, "1", "2", "3", "4", "5", convert=FALSE) %>% group_by(Week, Cluster)  
+```
+
+```{r}
+K6 <- K5 %>% summarize(Avg_Score=mean(Scores)) %>% print
+```
+```{r}
+K6$Week <- as.numeric(K6$Week)
+
+K6$Cluster <- as.factor(K6$Cluster) 
+
+```
+
+```{r}
+ggplot(K6, aes(x=Week, y=Avg_Score, color= Cluster))+
+  geom_line() +xlab("Week") + ylab("Average Motivation")
+```
+```{r}
+# Indiviudals in cluster 1, on average, have negative motivation scores as opposed to those in cluster 2. Furthermore, The average scores for both clusters are fluctuating. For cluster 1, there is an increase in the average motivation score in Week 3 and  Week 5. In cluser 2, however, there is an increase in the average motivation score in Week 2 and Week 4. In other words, if there is increase of motivation scores in cluster 1 during a particular week, there is a decrease of of motivation scores in cluster 2 in that week. 
+```
+
+```{r}
+# The number of people in each cluster
+(K7 <- count(K5, Cluster, sort=FALSE))
+```
+
+# 3 Clusters
+```{r}
+fit2 <- kmeans(K3, 3)
+fit2$cluster
+(K8 <- data.frame(K3, fit2$cluster))
+```
+
+```{r}
+names(K8) <- c("1", "2", "3", "4", "5", "Cluster")
+```
+
+      #Visualizing the 3 clusters. 
+```{r}
+K9 <- K8 %>% gather(Week,Scores, "1", "2", "3", "4", "5", convert=FALSE) %>% group_by(Week, Cluster)  
+```
+
+```{r}
+K10 <- K9 %>% summarize(Avg_Score=mean(Scores)) %>% print
+```
+
+```{r}
+K10$Week <- as.numeric(K10$Week)
+
+K10$Cluster <- as.factor(K10$Cluster) 
+```
+
+```{r}
+ggplot(K10, aes(x=Week, y=Avg_Score, color= Cluster))+
+  geom_line() +xlab("Week") + ylab("Average Motivation")
+```
+
+```{r}
+# The average motivation scores range from -1.5 to 1.25. For cluster 1, the average motivation score tends to be decreasing, with a slight increase in Week 3. Nonetheless, for clusters 2 and 3, the average motivation scores were fluctuating. However, starting from Week 3, there was an increasing trend of average motivation scores for cluster 2. Nevertheless, it can be said that there were negative average  motivation scores for cluster 3 and around zero for cluster 2. 
+```
+
+
+```{r}
+# The number of people in each cluster
+(K11 <- count(K9, Cluster, sort=FALSE))
+```
+```{r}
+# The cluster grouping with 3 clusters is more informative as the number of individuals is similar across the clusters or groups. Also, the line graph with three clusters paints a clearer picture of the distribution of motivation scores as opposed to one with two clusters. 
```

Comment:
Why does having similar sized groups mean that the clusters are more informative?